basics/registration.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Registration Form</title>
+</head>
+<body>
+<form id="registration">
+	<label for="input">Fill out the registration form below</label><br>
+	<label for="first name">First Name</label><br>
+	<input type="text" id="first name" name="first" /><br>
```

Comment:
id attribute should not contain whitespaces

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id#:~:text=id%20's%20value%20must%20not,have%20one%20single%20ID%20value.